packages/ag-charts-community/src/chart/interaction/syncManager.ts

```diff
@@ -1,13 +1,29 @@
-import type { Chart } from '../chart';
+import type { ChartAxisDirection } from '../chartAxisDirection';
+import type { ISeries } from '../series/seriesTypes';
 import { BaseManager } from './baseManager';
+import type { ZoomManager } from './zoomManager';
 
 type GroupId = string | symbol;
 
+/** Breaks circular dependencies which occur when importing ChartAxis. */
+type AxisLike = {
+    boundSeries: ISeries<any>[];
+    direction: ChartAxisDirection;
+    keys: string[];
+};
+
+/** Breaks circular dependencies which occur when importing Chart. */
+type ChartLike = {
```

Comment:
This seems to replicate `ChartService`. It's probably fine for now, but we might want to consider something like an `IChart` interface (similar to `ISeries` / `Series`)